house/models.py

```diff
@@ -0,0 +1,49 @@
+from django.db import models
+
+
+class Country(models.Model):
+    name = models.CharField(max_length=30)
+
+    def __str__(self):
+        return self.name
+
+
+class City(models.Model):
```

Comment:
Hey, are we sure we want it as a class? @shlior7 , I think it can be an enum(models.TextChoices) by code and not dynamic because we can know here what city we are going to include

Reply:
As I answered on slack, we need a City instance to know the country the city is in. that is not possible with TextChoices (If Im not mistaken) but only but making them a Class.